### DISCUSSION TITLE
Some ideas for the future

### DISCUSSION THREAD

**Discussion by tarsil** [dymmond/esmerald#198] [Ideas]
This will serve as the possible place where we can put some ideas for a roadmap.

- [x] Allow more native alternatives to Pydantic.
- [x] Handler signals (broadcasting).
- [ ] studying the possibility of adding lazy loading modules internally to improve performance.
- [x] Add support for msgspec

**Comment by devkral** [comment]
you may can just leave out the `__init__.py` files.
This way python treats the folders not as module and lazy loads for you.

You still can import via  folder.scriptname (without the .py of course).

**Reply by devkral** [reply]
I think in your projects, just removing the stub `__init__.py` files would work in most places without any other changes

**Reply by devkral** [reply]
sry, my idea was wrong. Empty `__init__.py` doesn't harm. I checked the docs again.
